### PR TITLE
Morphology level radial distance features use the soma as reference point

### PR DESCRIPTION
**Pull request by eleftherioszisis**
When a morphology object is used the radial distance features will use the soma as the reference point for calculating the distances, not the root of each neurite.

#982

### CODE REVIEW

File: neurom/features/morphology.py [hunk 1]
```diff
@@ -115,15 +115,77 @@ def soma_radius(morph):
...
+        neurite_type=neurite_type,
+        use_subtrees=use_subtrees
+    )))
+
+
+@feature(shape=(...,))
+def section_bif_radial_distances(
+    morph, origin=None, neurite_type=NeuriteType.all, use_subtrees=False
+):
+    """Get the radial distances of the bf sections."""
```
Comment (adrien-berchet) [inline neurom/features/morphology.py:163]:
```suggestion
    """Get the radial distances of the bifurcation sections."""
```

**Comment by eleftherioszisis** [comment]
@adrien-berchet , is this ok to merge?

**Review by adrien-berchet — APPROVED** [review]

**Comment by adrien-berchet** [comment]
> @adrien-berchet , is this ok to merge?

For me it is ok

**Comment by eleftherioszisis** [comment]
Thanks!

**Pull request merged**